### DISCUSSION TITLE
Black Hole suddenly stopped working, not sure why

### DISCUSSION THREAD

**Discussion by zolars1966** [ExistentialAudio/BlackHole#736] [Q&A] [answered]
Howdy, everyone! I'm trying to use BlackHole with Quicktime to record videos with my mic and system audio. But after a while, BlackHole just stopped working, tried reinstalling it, recreating aggregate and multi-output devices, but it didn't work. QuickTime records my microphone audio, but don't record system audio, though it did day ago. 
I tried to do something from similar discussions, but it didn't work. Help me please!
![Screenshot 2023-08-26 at 02 39 49](https://github.com/ExistentialAudio/BlackHole/assets/70763346/a119c6b3-d3ed-4259-9cd5-62948a53dd28)
![Screenshot 2023-08-26 at 02 39 58](https://github.com/ExistentialAudio/BlackHole/assets/70763346/3e13f29c-1d0e-4f5e-9bd7-c7ae0600030d)

**Comment by devinroth** [comment, marked as answer]
First off QuickTime doesn’t record aggregates. At least not in a way you’d expect.

Also drift correction is broken on the latest macOS release.

Do a test.

No aggregate/multi output. Output audio to BlackHole directly (you can set it as default output and play audio from YouTube). And record BlackHole using Audacity.

Also I recommend using BlackHole 2ch because of some odd macOS bugs.

Devin

**Reply by zolars1966** [reply]
Thanks a lot, helped!

**Reply by Ivanaxel1314** [reply]
> Thanks a lot, helped!
